specification/inference/put_googlevertexai/examples/request/PutGoogleVertexAiRequestExample10.yaml

```diff
@@ -0,0 +1,14 @@
+summary: A chat_completion task for Google Model Garden Mistral endpoint with single streaming URL provided
+description: Run `PUT _inference/chat_completion/google_model_garden_mistral_chat_completion` to create an inference endpoint to perform a `chat_completion` task using Mistral's model hosted on Google Model Garden with single streaming URL provided.
+method_request: 'PUT _inference/chat_completion/google_model_garden_mistral_chat_completion'
+# type: "request"
+value: |-
+  {
+      "service": "googlevertexai",
+      "service_settings": {
+          "provider": "mistral",
+          "model_id": "mistral-small-2503",
+          "service_account_json": "service-account-json",
+          "streaming_url": "https://url:streamRawPredict"
```

Comment:
Thanks for adding these examples. Is there anymore information we can give the user to help aid them in finding these URLs? Is there a general place where they can find them (like after creating the model garden endpoint, go to this tab to find the base url and you'll need to add `:streamRawPredict` to the end etc).

I'm not sure if the best place for that is here in the examples though.